src/sentry/api/endpoints/group_ai_autofix.py

```diff
@@ -0,0 +1,175 @@
+from __future__ import annotations
+
+import logging
+from datetime import datetime
+
+from django.conf import settings
+from django.http import HttpResponse
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.group import GroupEndpoint
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.models.commit import Commit
+from sentry.models.group import Group
+from sentry.models.release import Release
+from sentry.models.releasecommit import ReleaseCommit
+from sentry.models.repository import Repository
+from sentry.net.http import connection_from_url
+from sentry.tasks.ai_autofix import check_for_timeout
+from sentry.types.ratelimit import RateLimit, RateLimitCategory
+from sentry.utils import json
+
+logger = logging.getLogger(__name__)
+
+from rest_framework.request import Request
+
+seer_connection_pool = connection_from_url(settings.AI_AUTOFIX_URL, retries=0)
+
+TIMEOUT_SECONDS = 60 * 30  # 30 minutes
+
+
+@region_silo_endpoint
+class GroupAiAutofixEndpoint(GroupEndpoint):
+    publish_status = {
+        "POST": ApiPublishStatus.EXPERIMENTAL,
+        "GET": ApiPublishStatus.EXPERIMENTAL,
+    }
+    owner = ApiOwner.ML_AI
+    # go away
+    private = True
+    enforce_rate_limit = True
+    rate_limits = {
+        "POST": {
+            RateLimitCategory.IP: RateLimit(5, 1),
+            RateLimitCategory.USER: RateLimit(5, 1),
+            RateLimitCategory.ORGANIZATION: RateLimit(5, 1),
+        },
+        "GET": {
+            RateLimitCategory.IP: RateLimit(5, 1),
```

Comment:
Seems weird to have the same rate limit for GET requests as POST when only the POST request is expensive

Reply:
looks like we typically don't rate limit GET requests now that I looked at the other files, originally just copied this over from the POST request. Removed the limits